source/phoneInputHelpers.js

```diff
@@ -438,7 +438,7 @@ export function compare_strings(a, b) {
   // `localeCompare()` is available in latest Node.js versions.
   /* istanbul ignore else */
   if (String.prototype.localeCompare) {
-    return a.localeCompare(b);
+    return (a || b) ? (!a ? 1 : !b ? -1 : a.localeCompare(b)) : 0;
```

Comment:
What are the cases when `a` or `b` are `undefined`?
When considering this function only in the scope of this library.
Only country names are compared.
All default country names are present (this is checked at build time).
The only case when a country name might be non-present is when a developer uses their own `labels` property and new countries are added to `libphonenumber-js`.
In that case, I guess it would be better to substitute country name with country code.
Otherwise, a user might end up with empty select options which would be confusing.
I'll commit the "substitute country name with country code" part myself.

Reply:
Fair enough for me :)